xbmc/Application.cpp

```diff
@@ -2991,12 +2991,9 @@ void CApplication::OnPlayBackStarted(const CFileItem &file)
   // check if VideoPlayer should set file item stream details from its current streams
   if (file.GetProperty("get_stream_details_from_player").asBoolean()
       || ((!file.HasVideoInfoTag() || !file.GetVideoInfoTag()->HasStreamDetails())
-      && (file.IsDiscImage()
+      && (URIUtils::IsBluray(file.GetPath())
       || file.IsDVDFile()
-      || file.IsBDFile()
-      || file.IsBluray()
-      || file.IsDVD()
-      || file.IsOnDVD())))
```

Comment:
Why did you remove isDVD & IsOnDVD?

Reply:
Those checks aren't needed. If the file is on a DVD (either an actual physical one or on disk in a video_ts directory) then it either has to be a `.vob` file or an `.ifo`. They are the only playable files that can be in/on that media and `IsDVDFile()` will return true for them.
I don't see the point of checking if it's an actual physical DVD or that the file is on one when that is irrelevant.  All we need to know is that it is either `video_ts.ifo`, starts with `vts_` and either ends with `.vob` or `_0.ifo` or is `video_ts.vob`.

As that is all you can have on a dvd (we don't need to worry about the .bup files) then the check for `IsDVDFile()` is quite sufficient to identify dvd media regardless of the actual physical source, which we don't need for this case.

If the dvd is an `.iso` then that test will not detect it as such so `IsDiscImage()` is still needed for those cases.  However, if the disc image is a bluray, Kodi will have already identified it as such and the correct protocol will be present - if you have it enabled, the simplified bluray menu dialog will already be on screen before we get to these tests.